builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -837,6 +837,7 @@ def writeServerlessFile(config, environment_logical_id, accountDetails){
   }
 
   sh "sed -i -- 's/\${file(deployment-env.yml):service}/${configLoader.INSTANCE_PREFIX}-${config['domain']}-${config['service']}/g' serverless.yml"
+  sh "sed -i -- 's/\${file(deployment-env.yml):functionName}/${configLoader.INSTANCE_PREFIX}_${config['domain']}_${config['service']}_${env}/g' serverless.yml"
```

Comment:
This can simply be {functionName}. `deployment-env.yml` will never contain `functionName` anyways.

Reply:
I see that the other `sed` instructions use similar format but it's obsolete and should be fixed anyways.

Reply:
Address only `functionName`. Haven't changed the others.